contracts/lib/Organization.sol

```diff
@@ -83,16 +117,40 @@ contract Organization is OrganizationInterface, IsMemberInterface {
     /* Constructor */
 
     /**
-     * @notice The address that creates the contract is set as the initial
-     *         owner.
+     * @notice 
+     *
+     * @param _owner The address that shall be registered as the owner of the
+     *               organization.
+     * @param _admin The address that shall be registered as the admin of the
+     *               organization. Can be address(0) if no admin is desired.
+     * @param _workers An array of initial worker addresses. Can be an empty
+     *                 array if no workers are desired or known at construction.
+     * @param _expirationHeight If any workers are given, this will be the
+     *                          block height at which they expire.
      */
-    constructor() public
+    constructor(
+        address _owner,
+        address _admin,
+        address[] memory _workers,
+        uint256 _expirationHeight
+    )
+        public
     {
-        owner = msg.sender;
+        require(
+            _owner != address(0),
+            "The owner must not be the zero address."
+        );
+
+        owner = _owner;
+        admin = _admin;
+
+        for(uint256 i = 0; i < _workers.length; i++) {
+            setWorkerInternal(_workers[i], _expirationHeight);
+        }
     }
 
 
-    /* External Functions */
+    /* External functions */
 
     /**
      * @notice Proposes a new owner of this contract. Ownership will not be
```

Comment:
Let's remove return (success/failure) statement of the function.
Also, in other functions of the contract, we should remove the return statement if it is only success/failure.

Reply:
Why? It is currently used in various places throughout the [repo](https://github.com/OpenSTFoundation/mosaic-contracts/search?q=success_) and it is even part of the [style guide](https://github.com/OpenSTFoundation/openst-contracts/blob/2b16c4191455aea6ed974b0d0f543e99870f1277/SOLIDITY_STYLE_GUIDE.md#alignment).
I remember @deepesh-kn or @sarvesh-ost had an argument for it 🧐
Feedback please @deepesh-kn / @sarvesh-ost

Reply:
Let's discuss this point (/cc @deepesh-kn @sarvesh-ost @schemar)

I think, using both error handling mechanisms (exception & error code) should not be common case. In solidity world, as there is no way to differentiate what type of exception was thrown (no catching) occasionally using return error code concept can be useful to handle function based on return value. However, this is rare case, and we could mostly rely on exception to be thrown in case of error.

Reply:
Keeping success/failure helps in unit testing. `Method.call` in truffle tests return `true` or `false` which is more natural assert than expecting no exception. I would prefer keeping this as it is. This is being followed in the codebase. 
Also, I think this discussion is beyond the scope of this PR.  can this be discussed separately? 🤔If there is any action item, we can always create a separate ticket.  😃

Reply:
I always found it useful to be deterministic about what to expect from the function.
@pgev, we should discuss this on call to understand the advantage and disadvantage for this. For this PR we should not do changes in all the files,  we should create a new ticket for this after discussion.

Reply:
The only problem is that is has to be the same in `mosaic` and `openst`, so we have to make a decision for this function.